### DISCUSSION TITLE
help!   Error occurred when exporting data to Doris table using s3 load

### DISCUSSION THREAD

**Discussion by chenminghui-pacvue** [apache/doris#30394] [A - General / Q&A]
This is my configuration

LOAD LABEL pacvue_test.dayreportCampaign_all5
    (
        DATA INFILE("s3://cdc-log-us-v3/amazon/dbo/dayreportCampaign/20231201-164728805.parquet")
        INTO TABLE `dayreportCampaign_all`
        FORMAT AS "parquet"

    )
    WITH S3
    (
        "AWS_ENDPOINT" = "http://cdc-log-us-v3.s3.us-west-2.amazonaws.com",
        "AWS_ACCESS_KEY" = "xxxxx",
        "AWS_SECRET_KEY"="xxxxx",
        "AWS_REGION" = "us-west-2"

    )
    PROPERTIES
    (
        "timeout" = "3600"
    );

error   info

 WARN (pending-load-task-scheduler-pool-6|4854) [LoadJob.unprotectedExecuteCancel():655] LOAD_JOB=12338, transaction_id={0}, error_msg={Failed to execute load with error: errCode = 2, detailMessage = S3 list path exception. path=s3://cdc-log-us-v3/amazon/dbo/dayreportCampaign/20231201-164728805.parquet, err: errCode = 2, detailMessage = S3 list path failed. path=s3://cdc-log-us-v3/amazon/dbo/dayreportCampaign/20231201-164728805.parquet,msg=errors while get file status getFileStatus on s3://cdc-log-us-v3/amazon/dbo/dayreportCampaign/20231201-164728805.parquet: com.amazonaws.services.s3.model.AmazonS3Exception: Bad Request (Service: Amazon S3; Status Code: 400; Error Code: 400 Bad Request; Request ID: VN82V2HDEF4QDPP3; S3 Extended Request ID: b4Rl5GVUCXx6gewMb0pXhfMPSsfHXI9dvScznU+e3hkKzntmnQMfnpK4KaqyUiFpgqCcgllgg2dpW8TtV0jpnQ==; Proxy: null), S3 Extended Request ID: b4Rl5GVUCXx6gewMb0pXhfMPSsfHXI9dvScznU+e3hkKzntmnQMfnpK4KaqyUiFpgqCcgllgg2dpW8TtV0jpnQ==:400 Bad Request: Bad Request (Service: Amazon S3; Status Code: 400; Error Code: 400 Bad Request; Request ID: VN82V2HDEF4QDPP3; S3 Extended Request ID: b4Rl5GVUCXx6gewMb0pXhfMPSsfHXI9dvScznU+e3hkKzntmnQMfnpK4KaqyUiFpgqCcgllgg2dpW8TtV0jpnQ==; Proxy: null)}
![{5BBA90C0-6B0B-44ea-AF60-629999F904AD}](https://github.com/apache/doris/assets/153483652/0ab31db1-edf5-4ac2-81e8-26ea74583148)

**Comment by Sirivasv** [comment]
Can you try with:
```
 "AWS_ENDPOINT" = "http://s3.us-west-2.amazonaws.com"
```
As per the aws [docs](https://docs.aws.amazon.com/general/latest/gr/s3.html#s3_region) that seems to be the actual aws endpoint for s3.

**Reply by chenminghui-pacvue** [reply]
I have tried this before and also reported the same error

**Comment by lin-in** [comment]
hello，I have the same problem，Have you solved the problem？

**Reply by yagagagaga** [reply]
What's your doris version?
